### ISSUE TITLE
Trailing newline in script and style blocks

### ISSUE THREAD

**Issue by RedHatter** [sveltejs/prettier-plugin-svelte#222]
Tailing newlines are being added to the script and style blocks. This is quite annoying. 

For example
```html
<script>
  export let name;
</script>
```
becomes
```html
<script>
  export let name;

</script>
```
A minimal repository exemplifying the issue can be found [here](https://github.com/RedHatter/prettier-svelte-trailing-newline).
```
git clone https://github.com/RedHatter/prettier-svelte-trailing-newline
cd prettier-svelte-trailing-newline
npm install
npm run
```

**Comment by dummdidumm** [comment]
Yes this is a result of Prettier 2.3.0, I just committed a fix, release pending. In the meantime you can downgrade to Prettier 2.2.x .

**Comment by dummdidumm** [comment]
Those who use pnpm likely still observe this issue. The reason is that the workspace `prettier-plugin-svelte` version is likely not found (see #155) and that makes a reference check fail (see #235 for more details). #235 fixes this, release pending

**Comment by dummdidumm** [comment]
Version 2.3.1  which is also incoroprated into the latest Svelte for VS Code extension should fix this issue.
